great_tables/_locations.py

```diff
@@ -383,27 +417,114 @@ def _(loc: LocBody, data: GTData) -> list[CellPos]:
 # Style generic ========================================================================
 
 
+# LocHeader
+# LocTitle
+# LocSubTitle
+# LocStubhead
+# LocStubheadLabel
+# LocColumnLabels
+# LocColumnLabel
+# LocSpannerLabel
+# LocStub
+# LocRowGroupLabel
+# LocRowLabel
+# LocSummaryLabel
+# LocBody
+# LocSummary
+# LocFooter
+# LocFootnotes
+# LocSourceNotes
+
+
 @singledispatch
 def set_style(loc: Loc, data: GTData, style: list[str]) -> GTData:
     """Set style for location."""
     raise NotImplementedError(f"Unsupported location type: {type(loc)}")
 
 
+@set_style.register(LocHeader)
```

Comment:
Since we removed .locnum and just use the Loc isntances directly, we can consolidate most simple versions of `set_style()`